syntax = "proto3";

package AcFunDanmu.Im.Message;

import "User.proto";

message PullOldRequest {
	Im.Basic.User target = 1;
	int64 minSeq = 2;
	int64 maxSeq = 3;
	int32 count = 4;
	int64 targetId = 5;
	string strTargetId = 6;
}
